### ISSUE TITLE
Make it accessible for blind people

### ISSUE THREAD

**Issue by jmfergeau** [Anarios/return-youtube-dislike#1148]
### Extension or Userscript?

Extension

### Request or suggest a new feature!

One of my friends is blind and she needs a screen reader to access youtube and all. And she pointed me that the extension doesn't bring the number of dislikes to be readable by her screen reader. This is due to the dislike button missing the aria-label.

### Ways to implement this!

On the like button, you have an aria-label that says `"Click on &quot;I like&quot; for this video, like <number_of_likes> users"`. (Or whatever it is in english, i'm not english.) All the extension needs is to add a similar aria-label on the dislike button. Something like: `aria-label="Click on &quot;I don't like&quot; for this video, like <number_of_dislikes> user"` Should be easy to do, theoretically. I'd do it myself but i'm not sure of where to stick that in the code. ^^'

Thanks for them.

### Can you work on this?

- [x] Yes
- [ ] No

### Will you be available for follow-up questions to help developers implement this?

Yes
